quarkus-modules/quarkus-kogito/src/main/java/com/baeldung/kogito/boundary/model/LoanApplicationsInputDto.java

```diff
@@ -0,0 +1,10 @@
+package com.baeldung.kogito.boundary.model;
+
+import jakarta.validation.constraints.NotNull;
+
+public record LoanApplicationsInputDto(
+        @NotNull
+        LoanApplicationDto[] applications
```

Comment:
Only indent four spaces per level. I see similar issues in some of the other Java files.